docs/setup.rst

```diff
@@ -1,11 +1,22 @@
 Setup
 =====
 
-TODO: move from README
-
-.. Indices and tables
-.. ==================
-..
-.. * :ref:`genindex`
-.. * :ref:`modindex`
-.. * :ref:`search`
+To get started, clone this repo...
+
+.. code-block::
+
+    git clone https://github.com/neurodatawithoutborders/nwb_benchmarks.git
+    cd nwb_benchmarks
+
+Setup a completely fresh environment...
+
+.. code-block::
+
+    conda env create --file environments/nwb_benchmarks.yaml --no-default-packages
+    conda activate nwb_benchmarks
+
+Setup initial machine configuration values with
```

Comment:
We should also add a link on how to install `tshark` . I think the `environments/nwb_benchmarks.yaml` only installs `pyshark` but `tshark` needs to be installed separately .

Reply:
Yeah that was needed on both Windows and Mac (a tad easier on Mac)

Added!